### PR TITLE
meta-package-manager: Add version 5.14.2

### LINKED ISSUE #5448: [Request]: Meta Package Manager
### Prerequisites

- [X] I have searched all issues/PRs to ensure it has not already been reported or fixed.

### Criteria

- [X] Non-GUI tool
- [X] Reasonably well-known and widely used (e.g. if it's a GitHub project, it should have at least 500 stars and/or 150 forks)
- [X] English interface (or at least English documentation)
- [X] Latest stable version
- [X] Full version (i.e. not a trial version)
- [X] Fairly standard install (e.g. uses a version-specific download URL, no elaborate pre/post install scripts)

### Name

meta-package-manager

### Description

wraps all package managers with a unifying CLI

### Homepage

https://kdeldycke.github.io/meta-package-manager/
https://github.com/kdeldycke/meta-package-manager

### Download Link(s)

https://github.com/kdeldycke/meta-package-manager/releases/download/v5.14.2/mpm-windows-x64.exe

### Some Indication of Popularity/Repute

420 ⭐

### PR DESCRIPTION
**Pull request by wickles**
Closes #5448

- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

File: bucket/meta-package-manager.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+        }
+    },
+    "bin": [ [ "mpm-windows-x64.exe", "mpm" ] ],
+    "checkver": {
+        "github": "https://github.com/kdeldycke/meta-package-manager"
+    },
+    "autoupdate": {
+        "architecture": {
+            "64bit": {
+                "url": "https://github.com/kdeldycke/meta-package-manager/releases/download/v$version/mpm-windows-x64.exe"
```
Comment (rashil2000) [inline bucket/meta-package-manager.json:19]:
```suggestion
                "url": "https://github.com/kdeldycke/meta-package-manager/releases/download/v$version/mpm-windows-x64.exe#/mpm.exe"
```

File: bucket/meta-package-manager.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    "description": "Wraps all package managers with a unifying CLI",
+    "homepage": "https://kdeldycke.github.io/meta-package-manager/",
+    "license": "GPL-2.0-only",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/kdeldycke/meta-package-manager/releases/download/v5.14.2/mpm-windows-x64.exe",
+            "hash": "894a1c0cc2dabd485f16869f6396f524f9fdf609a66f9915df8e454048130710"
+        }
+    },
+    "bin": [ [ "mpm-windows-x64.exe", "mpm" ] ],
```
Comment (rashil2000) [inline bucket/meta-package-manager.json:12]:
```suggestion
    "bin": "mpm.exe",
```

**Review by rashil2000 — COMMENTED** [review]
Also I think we can rename the manifest to just mpm.json

**Comment by kdeldycke** [comment]
Thanks @wickles for adding Meta Package Manager to the main Scoop repository! 🥳

**Pull request merged**